Makefile

```diff
@@ -112,13 +112,68 @@ up.jason:
 	npm run start --prefix=jason/e2e-demo
 
 
-# Run Medea media server in development mode.
+# Run Medea media server.
 #
 # Usage:
-#	make up.medea
+#	make up.medea [dockerized=(NO|yes)] [background=(NO|yes)
+#	              [logs=(NO|yes)]] [TAG=(dev|<docker-tag>)]
+#                 [registry=(dockerhub|<custom-registry>)]
+
+docker-image-name = $(if $(call eq,$(registry),),,$(registry)/)$(MEDEA_IMAGE_NAME)
+docker-up-tag = $(if $(call eq,$(TAG),),dev,$(TAG))
 
 up.medea:
+ifeq ($(dockerized),yes)
+	@make down.medea dockerized=yes
+	COMPOSE_IMAGE_NAME=$(docker-image-name) \
+	COMPOSE_IMAGE_VER=$(docker-up-tag) \
+	docker-compose -f docker-compose.medea.yml up \
+		$(if $(call eq,$(background),yes),-d,--abort-on-container-exit)
+ifeq ($(background),yes)
+ifeq ($(logs),yes)
+	docker-compose -f docker-compose.medea.yml logs -f
+endif
+endif
+else
+	@make down.medea dockerized=no
 	cargo run --bin medea
+endif
+
+
+
+
+#####################
+# Stopping commands #
+#####################
+
+# Stop all related to Medea services.
+
+down:
+	@make down.medea dockerized=yes
+	@make down.medea dockerized=no
+	@make down.coturn
+
+
+# Stop Medea media server.
+#
+# Usage:
+# 	make down.medea [dockerized=(NO|yes)]
```

Comment:
There is no sense to highlight defaults with uppercase. According to docopt format (or at lease our variant of its usage) defaults always go first in declaration.